Sources/FoundationExtensions/Locale+Extensions.swift

```diff
@@ -29,7 +36,7 @@ extension Locale {
         #endif
     }
 
-    // swiftlint:disable:next identifier_name
+    /// Returns the language code using the new API if available, falling back to the deprecate API
```

Comment:
Similar comment here (inspired by Apple's documentation for `languageCode`)
```suggestion
    /// The language code that identifies the locale's language.
```